pom.xml

```diff
@@ -40,14 +40,34 @@
         <module>nihms-cli</module>
     </modules>
 
+    <profiles>
+
+        <profile>
+            <id>external</id>
+            <activation>
+                <property>
+                    <name>external</name>
+                </property>
+            </activation>
+            <properties>
+                <scp.port>122</scp.port>
```

Comment:
Did you mean for this SCP port to be 122?  Below it is 22, which is the regular port for SCP.

Reply:
@Trumbore yeah, good catch.

The profile name `external` derives from the fact that the `scp.port` needs to be modified when deploying Maven artifacts from outside of the JHU network.

Deployment of Maven artifacts happens over `scp`.

JHU blocks incoming SSH/SCP traffic from outside the institution; connections to port 22 originating outside JHU's network will get dropped, unless you have made special arrangements with network security.  Instead, SSH/SCP connections originating outside of JHU require that you connect to port `122`, instead.

So this Maven profile, when activated with the `-Dexternal` property, will use port `122` to initiate SCP connections to JHU, instead of using port `22`.